source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingRecords.sql

```diff
@@ -100,6 +101,7 @@ BEGIN TRY
 		,p.NhsNumber										AS NhsNumberToLookup --this will later be reformatted if valid
 		,p.GivenName										AS Forename
 		,p.FamilyName										AS Surname
+		,LEFT(p.GivenName,1)+'.'+LEFT(p.FamilyName,1)+'.'   AS Initials
```

Comment:
Not sure I would have gone as far as storing the initials in SQL server but as you have, I would just store the initials with no dots between them.